scala-package/core/src/main/scala/org/apache/mxnet/util/NativeLibraryLoader.scala

```diff
@@ -88,6 +88,12 @@ private[mxnet] object NativeLibraryLoader {
     saveLibraryToTemp("libmxnet.so", "/lib/native/libmxnet.so", true)
     saveLibraryToTemp("libgfortran.so.3", "/lib/native/libgfortran.so.3", false)
     saveLibraryToTemp("libquadmath.so.0", "/lib/native/libquadmath.so.0", false)
+    saveLibraryToTemp("libiomp5.so", "/lib/native/libiomp5.so", false)
+    saveLibraryToTemp("libiomp5.dylib", "/lib/native/libiomp5.dylib", false)
+    saveLibraryToTemp("libmklml_intel.so", "/lib/native/libmklml_intel.so", false)
+    saveLibraryToTemp("libmklml.dylib", "/lib/native/libmklml.dylib", false)
+    saveLibraryToTemp("libmkldnn.so.0", "/lib/native/libmkldnn.so.0", false)
+    saveLibraryToTemp("libmkldnn.0.dylib", "/lib/native/libmkldnn.0.dylib", false)
```

Comment:
Is there a way we can obtain every file name in the native folder and apply them here?
```
https://stackoverflow.com/questions/5694385/getting-the-filenames-of-all-files-in-a-folder
```

Reply:
There actually might be unnecessary .so files. If the script didn't clean them, we might accidentally includes some garbage in the jar.

It's much safer to explicitly include files that we need. And it's also a good for documentation purpose.